develop-native/solutions/expense-function/src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
-quarkus.native.builder-image=registry.ocp4.example.com:8443/redhattraining/ubi-quarkus-mandrel-builder-image:jdk-21
+quarkus.native.builder-image=registry.ocp4.example.com:8443/quarkus/ubi-quarkus-mandrel-builder-image:jdk-21
```

Comment:
Why this change?

Reply:
Because the script that syncs the images uses the same directory as the upstream one to replicate.

Reply:
Cool, nice catch!